### ISSUE TITLE
Show charging/discharging permissions for battery

### ISSUE THREAD

**Issue by mr-manuel** [victronenergy/gui-v2#516]
Display in the "Overview" page on the battery symbol, if `com.victronenergy.battery.*/Io/AllowToCharge` or `com.victronenergy.battery.*/Io/AllowToDischarge` is `0`. This values should be read from the controlling BMS which is found in `com.victronenergy.system/ActiveBatteryService`.

- Display the upper part of the battery in a redish color, if `AllowToCharge` is `0`
- Display the lower part of the battery in a redish color, if `AllowToDischarge` is `0`

Related Slack thread: https://victrondevelopment.slack.com/archives/C04RYU0UMDE/p1689670838212369

**Comment by mpvader** [comment]
on hold, discussion on what we want to do continues a bit.

**Comment by mpvader** [comment]
and @mr-manuel hope its fine with you, but I removed your design to make sure we won't have any confusion on what needs to be implemented - once the design is ready.

**Comment by chriadam** [comment]
This isn't for 1.0.0 I believe, marking as low_prio.

**Comment by mpvader** [comment]
Indeed, not for v1

**Comment by chriadam** [comment]
This is going to be important for ESS stuff, I assume @philipptrenz
